File: screenshots/capture.py
from playwright.sync_api import sync_playwright

PORT = 5175
W, H = 480, 860

with sync_playwright() as p:
    browser = p.chromium.launch()
    ctx = browser.new_context(viewport={"width": W, "height": H}, device_scale_factor=2)
    page = ctx.new_page()
    page.goto(f"http://localhost:{PORT}", wait_until="networkidle")
    page.wait_for_timeout(500)

    page.screenshot(path="raw/tune.png")
    print("✓ tune.png")

    page.get_by_role("button", name="Record").click()
    page.wait_for_timeout(300)
    page.screenshot(path="raw/recording.png")
    print("✓ recording.png")

    page.get_by_role("button", name="System").click()
    page.wait_for_timeout(300)
    page.screenshot(path="raw/system.png")
    print("✓ system.png")

    browser.close()
    print("Done")

File: screenshots/capture.mjs
import { chromium } from 'playwright';

const PORT = 5175;
const W = 390, H = 844;

const browser = await chromium.launch();
const ctx = await browser.newContext({ viewport: { width: W, height: H }, deviceScaleFactor: 2 });
const page = await ctx.newPage();
await page.goto(`http://localhost:${PORT}`, { waitUntil: 'networkidle' });
await page.waitForTimeout(500);

// Tune (default, full page)
await page.screenshot({ path: 'raw/tune.png', fullPage: true });

// Recording
await page.getByRole('button', { name: 'Recording' }).click();
await page.waitForTimeout(300);
await page.screenshot({ path: 'raw/recording.png', fullPage: true });

// System
await page.getByRole('button', { name: 'System' }).click();
await page.waitForTimeout(300);
await page.screenshot({ path: 'raw/system.png', fullPage: true });

await browser.close();
console.log('Done: tune.png, recording.png, system.png');
